charts/portal/values.yaml

```diff
@@ -618,13 +622,17 @@ backend:
         cpu: 75m
         memory: 350M
     seeding:
-      testDataEnvironments: ""
+      # -- If set to true the data configured in the config map 'configmap-backend-seeding-initialdata.yaml' will be taken to insert the initial data
+      useInitial: false
       # -- Option to seed test data provided in a configMap
-      testData:
-        # -- ConfigMap containing json files for the tables to seed, e.g. companies.test.json, addresses.test.json, etc.
-        configMap: ""
-        # -- Filename identifying the test data files e.g. for companies.test.json the value would be "test"
-        filename: ""
+      seedTestData:
+        # -- If set to true the data configured in the config map 'configmap-backend-seeding-testdata.yaml' will be taken to insert the default test data
```

Comment:
Please update comment to [here](https://github.com/SAP/ssi-dim-middle-layer/blob/main/charts/dim/values.yaml#L66)